src/main/java/ua/softserveinc/tc/util/ApplicationConfiguratorPropertiesBased.java

```diff
@@ -238,4 +246,35 @@ public Integer getHourToSendEmailReminder() {
     public Integer getMinutesToSendEmailReminder() {
         return minutesToSendEmailReminder;
     }
+
+    /**
+     * Receives a String of acceptable image formats and returns
+     * a String array of normalized image formats. The String of acceptable
+     * image formats must contains image extensions separated by
+     * whitespace any length. Method handles following image extension:
+     * .jpeg .jpg .jpe .jfif .jif .jfi .png .tiff .tif .bmp .dib. .gif
+     *
+     * @param formatString String of acceptable image formats
+     * @return String array of normalized image formats
+     */
+    private String[] ImageFormatNormalizator(String formatString) {
+        String[] arraysFormats = formatString.trim().split("\\s");
+        for (int i = 0; i < arraysFormats.length; i++)
+            switch (arraysFormats[i].toLowerCase()) {
+                case "jpg":
+                case "jpe":
+                case "jfif":
+                case "jif":
+                case "jfi":
+                    arraysFormats[i] = "jpeg";
+                    break;
+                case "tif":
+                    arraysFormats[i] = "tiff";
+                    break;
+                case "dib":
+                    arraysFormats[i] = "bmp";
+                    break;
+            }
+        return  arraysFormats;
```

Comment:
This is algorithm for normalization. png not need this, because it have only one file extension - .png.